src/main/java/org/opensearch/jobscheduler/JobSchedulerPlugin.java

```diff
@@ -5,6 +5,15 @@
 
 package org.opensearch.jobscheduler;
 
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionResponse;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.cluster.node.DiscoveryNodes;
+import org.opensearch.common.settings.*;
```

Comment:
Don't use star imports.  (You probably want to disable them on your IDE)

Reply:
done